book/src/404.md

```diff
@@ -0,0 +1,3 @@
+# Document not found (404)
+
+Apologies: This URL is invalid. Try the search instead.
```

Comment:
This page does not appear to be used, remove.

Reply:
Resolving as can't fix in 3-6 months.